packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -110,6 +110,15 @@ describe("<DateRangePicker>", () => {
             assert.equal(dateRangePicker.state.displayMonth, NOVEMBER);
         });
 
+        it("is value - 1 if set and initialMonth not set and value month === maxDate month", () => {
```

Comment:
hrmm this description is more confusing than helpful. reword please to describe the issue being tested, not the code itself.